pkg/logging/logging.go

```diff
@@ -0,0 +1,256 @@
+// Copyright 2021 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package fybriklogging
+
+import (
+	"encoding/json"
+
+	"github.com/go-logr/logr"
+)
+
+// LogType indicates the types of log entries supported.  Logging packages typically provide separate function
+// calls for the different log types.
+type LogType string
+
+const (
+	ERROR LogType = "error" // Errors encountered in the code
+	INFO  LogType = "info"  // Info logs are things you want to tell the user or administrator which are not errors.
+)
+
+// VerbosityLevel allows one to decide how "chatty" you want your logs to be.
+type VerbosityLevel int
+
+const (
+	MANDATORY VerbosityLevel = 0
```

Comment:
Why not just int? It will be less conversions to make...

Reply:
@shlomitk1 True, but it's less clear when reading the code.

Reply:
OK.